From b40d01c59b453358c09e72da4f4fd1d713d1a610 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Wed, 29 Apr 2015 15:17:09 +0300
Subject: [PATCH 0642/1240] fix: i2c: extend i2c help message to be more
 informative

	extend 'i2c' command help message to be more informative
	regarding the arguements:
	- address.[.0, .2] = Address length [byte, 2byte]
	- xx.[.b, .w, .l]  = Returned Data length [byte, 2byte, 4byte]

Change-Id: Ic84a8e8511653d03a7f6f3a61c7bfd2d6ab24b2d
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26598
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 common/cmd_i2c.c | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/common/cmd_i2c.c b/common/cmd_i2c.c
index 22db1bb..f3838ba 100644
--- a/common/cmd_i2c.c
+++ b/common/cmd_i2c.c
@@ -1872,6 +1872,8 @@ static int do_i2c(cmd_tbl_t * cmdtp, int flag, int argc, char * const argv[])
 /***************************************************/
 #ifdef CONFIG_SYS_LONGHELP
 static char i2c_help_text[] =
+	"address.[.0, .2] = Address length [byte, 2byte]"
+	"xx.[.b, .w, .l]  = Returned Data length [byte, 2byte, 4byte]"
 #if defined(CONFIG_SYS_I2C)
 	"bus [muxtype:muxaddr:muxchannel] - show I2C bus info\n"
 #endif
@@ -1885,9 +1887,9 @@ static char i2c_help_text[] =
 #endif  /* CONFIG_I2C_EDID */
 	"i2c loop chip address[.0, .1, .2] [# of objects] - looping read of device\n"
 	"i2c md chip address[.0, .1, .2] [# of objects] - read from I2C device\n"
-	"i2c mm chip address[.0, .1, .2] - write to I2C device (auto-incrementing)\n"
-	"i2c mw chip address[.0, .1, .2] value [count] - write to I2C device (fill)\n"
-	"i2c nm chip address[.0, .1, .2] - write to I2C device (constant address)\n"
+	"i2c mm[.b, .w, .l] chip address[.0, .1, .2] - write to I2C device (auto-incrementing)\n"
+	"i2c nm[.b, .w, .l] chip address[.0, .1, .2] - write to I2C device (constant address)\n"
+	"i2c mw chip address[.0, .1, .2] value [count] - write to I2C device (fill constant value)\n"
 	"i2c probe [address] - test for and show device(s) on the I2C bus\n"
 	"i2c read chip address[.0, .1, .2] length memaddress - read to memory\n"
 	"i2c write memaddress chip address[.0, .1, .2] length - write memory to i2c\n"
-- 
1.9.1

